source/Calamari.Azure.CloudServices/Deployment/Conventions/DeployAzureCloudServicePackageConvention.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System.IO;
 using System.Reflection;
 using Calamari.Commands.Support;
+using Calamari.Common.Variables;
 using Calamari.Deployment;
 using Calamari.Deployment.Conventions;
 using Calamari.Integration.EmbeddedResources;
 using Calamari.Integration.FileSystem;
 using Calamari.Integration.Processes;
 using Calamari.Integration.Scripting;
+using SpecialVariables = Calamari.Deployment.SpecialVariables;
```

Comment:
This is a smell. What is the difference between `DeploymentVariables`, `Calamari.Deployment.SpecialVariables` and `Calamari.Common.Variables.SpecialVariables`? Perhaps they need a clear name.

Or consider using inheritance like in https://github.com/OctopusDeploy/Calamari/pull/539/files#diff-d879eb9b732251cb70f59aec282b91fcR150 and https://github.com/OctopusDeploy/Calamari/pull/539/files#diff-d879eb9b732251cb70f59aec282b91fcR9